typescript/sdk/src/deploy/verify/utils.test.ts

```diff
@@ -0,0 +1,54 @@
+import { describe, expect } from '@jest/globals';
+
+import { ChainMap, ChainName } from '../../types.js';
+
+import { ContractVerificationInput } from './types.js';
+import { shouldAddVerificationInput } from './utils.js';
+
+describe('shouldAddVerificationInput', () => {
+  it('should return true if the artifact does not exist in the verification inputs', () => {
+    const verificationInputs: ChainMap<ContractVerificationInput[]> = {
+      Ethereum: [
+        {
+          name: 'ContractA',
+          address: '0x123',
```

Comment:
eqAddress may fail if it's not a proper eth address - can grab `randomAddress()` from testUtils.ts

ditto below